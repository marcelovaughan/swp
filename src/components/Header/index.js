import React from 'react';

export default function Header() {
    return (        
        <header>
            <h1 className="logo">StarWars - Planets</h1>
            <h2>PLANETS</h2>
        </header>
    );
}