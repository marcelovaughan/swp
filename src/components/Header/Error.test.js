import React from "react"
import { configure, mount } from "enzyme"
import Adapter from "enzyme-adapter-react-16"
import Header from "./index";

configure({ adapter: new Adapter() });

describe("Header Render", () => {
    it("Render the component", () => {
        const component = mount(<Header />);
        expect(component.debug()).toMatchSnapshot();
    });
});
