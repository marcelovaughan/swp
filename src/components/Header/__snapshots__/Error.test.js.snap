// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Render Render the component 1`] = `
"<Header>
  <header>
    <h1 className=\\"logo\\">
      StarWars - Planets
    </h1>
    <h2>
      PLANETS
    </h2>
  </header>
</Header>"
`;
